lib/ansible/module_utils/urls.py

```diff
@@ -308,6 +309,11 @@ def __init__(self, *args, **kwargs):
                 self.context = create_default_context()
             elif HAS_URLLIB3_PYOPENSSLCONTEXT:
                 self.context = PyOpenSSLContext(PROTOCOL)
+
+            # Support insecure TLS protocol options
+            # if self.context and self.tls_insecure:
+            #     tls_insecure_options(self.context)
```

Comment:
It appears to support this it will require adding an `__init__` method to `CustomHTTPSHandler` so that `tls_insecure` can be stored and passed to `CustomHTTPSConnection`.  See `HTTPSClientAuthHandler` for an example of what would be required.

Reply:
This is now implemented.